client/src/Utils/ControllerHooks/useExposuresSaving.ts

```diff
@@ -63,7 +90,7 @@ const useExposuresSaving = () => {
             filteredExposures = (filteredExposures as Exposure[]).map(extractExposureData);
         }
         saveExposureAndFlightDataLogger.info('launching the server request', Severity.LOW);
-
+        checkUpdateInvestigationReasonId(filteredExposures)
         return axios.post('/exposure/updateExposures', {
```

Comment:
is update exposures dependent on the reason Id?

Reply:
no